src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/EvalResultDataItem.cs

```diff
@@ -21,13 +33,13 @@ namespace Microsoft.CodeAnalysis.ExpressionEvaluator
     /// </remarks>
     internal sealed class EvalResultDataItem : DkmDataItem
     {
-        /// <summary>
-        /// May be null for synthesized nodes like "Results View", etc.
-        /// </summary>
-        public readonly string NameOpt;
+        public readonly ExpansionKind Kind;
```

Comment:
And is Kind used to set { Name, Value, TypeName }? If so, can those be fields on EvalResultDataItem that are set explicitly by each GetRows implementation?